### ISSUE TITLE
bug: Create Plugin App /dist Folder Missing

### ISSUE THREAD

**Issue by 0xJonHoldsCrypto** [rabbitholegg/questdk-plugins#400]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

2.0.0-alpha.50

### Current Behavior

When running `pnpm run create` in the /apps/create-plugin folder it appears that a folder is missing or obsolete resulting in an error. 
![image](https://github.com/rabbitholegg/questdk-plugins/assets/81202085/d919a41e-2013-4cca-9bab-5d62f43fcc16)

### Expected Behavior

Template folder creation after running `pnpm run create`

### Steps To Reproduce

Current version clone, build, navigate to apps/create-plugin, run `pnpm run create`

### Anything else?

Looking to add DEMOS actions to the questSDK

**Comment by mmackz** [comment]
You need to run some of the build steps first.

Or you can run `pnpm run create` from root and it should run the build step before starting the plugin builder

**Comment by 0xJonHoldsCrypto** [comment]
I had already done that. 
A re-run gives the following issues - all of which are replicated by two other members on my team
Thanks in advance!
![image](https://github.com/rabbitholegg/questdk-plugins/assets/81202085/8931e5bb-a633-49e4-a064-592c812060ed)
